src/braket/devices/devices.py

```diff
@@ -50,7 +50,7 @@ class _Rigetti(str, Enum):
         _AspenM1 = "arn:aws:braket:us-west-1::device/qpu/rigetti/Aspen-M-1"
         _AspenM2 = "arn:aws:braket:us-west-1::device/qpu/rigetti/Aspen-M-2"
         _AspenM3 = "arn:aws:braket:us-west-1::device/qpu/rigetti/Aspen-M-3"
-        Ankaa2 = "arn:aws:braket:us-west-1::device/qpu/rigetti/Ankaa-2"
+        _Ankaa2 = "arn:aws:braket:us-west-1::device/qpu/rigetti/Ankaa-2"
```

Comment:
Will this break tests for notebooks that are still referencing Ankaa-2?

Reply:
Only if the pinned SDK version is updated; we can bump up the version in the notebooks after we make the corresponding changes there.